### PR TITLE
Create org.emilien.SpaceLaunch

### PR DESCRIPTION
**Pull request by elescoute**
Initial version

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.emilien.SpaceLaunch [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/elescoute/spacelaunch",
+                    "commit": "3358513e1c62a888f01ca4fdfe4cd98cf45e2ef4"
+                }
+            ]
+        }
+    ],
+    "build-options" : {
+        "env" : {        }
```
Comment (nedrichards) [inline org.emilien.SpaceLaunch:40]: should be removed if this is really empty

**Comment by nedrichards** [comment]
bot, build org.emilien.SpaceLaunch

**Comment by nedrichards** [comment]
your file doesn't have an extension. it needs to be .json if it's a JSON file for the buildbot to pick it up correctly.

File: org.emilien.SpaceLaunch [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "spacelaunch",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/elescoute/spacelaunch",
```
Comment (hfiguiere) [inline org.emilien.SpaceLaunch:33]: when I try to view that project on gitlab, after being asked to login, it's 404. So either it's not the right URL or it is private and in either case, the build will fail. Also usually the checkout url end in `.git`

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
also do you own that domain? It resolve but the root in `404 not found`

**Comment by nedrichards** [comment]
bot, build org.emilien.SpaceLaunch

**Comment by nedrichards** [comment]
bot, build org.emilien.SpaceLaunch

**Comment by hfiguiere** [comment]
bot, build org.emilien.SpaceLaunch

**Comment by hfiguiere** [comment]
appstream has both an appdata and metainfo. There should be one, metainfo being the new name.

The app-id in either is wrong and doesn't match the id.

This would explain the build failure.

**Comment by nedrichards** [comment]
bot, build org.emilien.SpaceLaunch

**Comment by elescoute** [comment]
It's ready for review.

**Pull request closed without merging**